contrib/bump_version.bash

```diff
@@ -6,13 +6,19 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR
 cd ..
 
-NEXT_VERSION=`cat .evg.yml | grep cli_version: | awk '{ print $2 }' | tail -n 1`
+BUMP_TYPE=$1
+if [ "$BUMP_TYPE" != "patch" ] && [ "$BUMP_TYPE" != "minor" ] && [ "$BUMP_TYPE" != "major" ]; then
+	echo $"Usage: $0 <patch|minor|major>"
+	exit 1
+fi
 
 LAST_VERSION=`node -e 'console.log(require("./package.json").version)'`
 
-echo "Bumping CLI v$LAST_VERSION to v$NEXT_VERSION"
+echo "Bumping CLI v$LAST_VERSION to"
```

Comment:
to.... WHERE!!!

Reply:
😆  ... so the first thing `npm version` does is print `v$NEXT_VERSION`.  The linebreak b/w versions is annoying, but I wanted to print something _before_ the command is run and we rely on the command to know what that next version is (this also gets rid of the `v$NEXT_VERSION` stutter)